devops: support `npx playwright install --with-deps` command

aslushnikov opened the pull request:
The common pattern today is to have two commands to configure CI:

```sh
npx playwright install-deps chromium
npx playwright install chromium
```

With this patch, this becomes:

```sh
npx playwright install --with-deps chromium
```

Note: `--with-deps` might call `sudo`.

yury-s reviewed (approved):
Consider installing deps by default and overriding that behavior with --no-deps

aslushnikov committed:
devops: support `npx playwright install --with-deps` command

The common pattern today is to have two commands to configure CI:

```sh
npx playwright install-deps chromium
npx playwright install chromium
```

With this patch, this becomes:

```sh
npx playwright install --with-deps chromium
```

Note: `--with-deps` might call `sudo`.

aslushnikov committed:
fix opts
